### ISSUE TITLE
MountVolume.SetUp failed for volume "pv-blobfuse-flexvol" : invalid character 'E' looking for beginning of value

### ISSUE THREAD

**Issue by sshehata** [Azure/kubernetes-volume-drivers#69]
**What happened**:
Recreating an AKS cluster with an newer version of Kubernetes does not work.
Storage account credentials and container name were working on the cluster before recreating it for the update (Before that cluster has been up for a long time).

**What you expected to happen**:
Successfully mount the azure blob storage container to the pod. Pod should end up in ready state.

**How to reproduce it**:

**Anything else we need to know?**:
/var/log/blobfuse-driver.log
[blobfuse-driver.log](https://github.com/Azure/kubernetes-volume-drivers/files/4884246/blobfuse-driver.log)

blobfuse-flexvol-installer.log
[blobfuse-flexvol-installer.log](https://github.com/Azure/kubernetes-volume-drivers/files/4884252/blobfuse-flexvol-installer.log)

When following instructions on the aks nodes, mounting is successful:
```

su
mkdir test
export AZURE_STORAGE_ACCOUNT=dixu0test
export AZURE_STORAGE_ACCESS_KEY=xxx
blobfuse mount test --container-name=dixu-test --tmp-path=/tmp/blobfuse -o allow_other --file-cache-timeout-in-seconds=120

```
If the same commands are done without `su`, it complains about the allow_other option

**Environment**:
- Kubernetes version (use `kubectl version`): 1.17.5
- OS (e.g. from /etc/os-release): ubuntu 16.04
- Kernel (e.g. `uname -a`): Linux aks-nodepool1-34600910-vmss000000 4.15.0-1089-azure #99~16.04.1-Ubuntu SMP Fri Jun 5 15:30:32 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
- Install tools: blobfuse-flexvol-installer-1.9.yaml 
- Others:

EDIT:
When creating a bigger cluster with many of the pods that mount the blob container, some of the pods actually do successfully mount it while most still show the above error.

**Comment by andyzhangx** [comment]
could be related to this issue: https://github.com/Azure/kubernetes-volume-drivers/issues/66#issuecomment-649188681

**Comment by sshehata** [comment]
I can confirm that the workaround mention here: #66 works.

**Comment by andyzhangx** [comment]
AKS has already rolled out a fix which would install blobfuse 1.1.1 on agent node, that won't be an issue soon.
